docker/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN apt-get update && apt-get install -y \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Disable IPv6 (https://rvm.io/rvm/security#ipv6-issues)
+RUN mkdir -p ~/.gnupg && echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
```

Comment:
Is your intention to have it both for CMake and LLVM?

Reply:
Yes, I see no downside to this restriction.

Reply:
I am hesitant to merge this.  Do we have evidence that IPv6 support is disabled on the docker daemons on slave machines?

Reply:
No. But even it is enabled, it is safe as IPv4 is definitely enabled, and all gpg queries will be passed through that.

Reply:
For the record, running this PR without this line resulted in the error:
```
gpg: keyserver receive failed: Cannot assign requested address
```

Reply:
How do you explain that is not consistently failing?

Reply:
If you mean the code that is in master, that is likely time outs. I was not able to download the keys from the specified keyserver using command line either, but after removing keyserver it was fine.

I also saw that the keyserver before that (sks) is/was under flood attack [here](https://salsa.debian.org/debian/gnupg2/blob/93daa0064ee29372d93661cc6a9aa990f8c52414/debian/NEWS)
```
gnupg2 (2.2.17-1) unstable; urgency=medium

Upstream GnuPG now defaults to not accepting third-party certifications from
the keyserver network. Given that the SKS keyserver network is under attack
via certificate flooding, and third-party certifications will not be
accepted anyway, we now ship with the more tightly-constrained and abuse-
resistant system hkps://keys.openpgp.org as the default keyserver.
```